packages/gamut-labs/src/experimental/Card/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { Box } from '@codecademy/gamut';
+import { Box } from '@codecademy/gamut/src';
```

Comment:
this should be `'@codecademy/gamut'`, that may be why you're seeing some typescript compile errors.

```suggestion
import { Box } from '@codecademy/gamut';
```

Reply:
ah yes, i changed this b/c i was seeing errors but once i rebuilt, the error went away.